.github/workflows/main.yml

```diff
@@ -1,13 +1,17 @@
 name: Main test
 
-on: [push, pull_request]
+on:
+  push:
+    branches:
+      - 'master'
+  pull_request:
 
 jobs:
   build-dummy-app-webpack-test-bundles:
     strategy:
       matrix:
         ruby: [2.7, 3.2]
-        node: [16, 18]
+        node: [14, 16, 18]
```

Comment:
Is there any particular reason for bringing back Node 14?
Before we decided to drop support for Node 14. (Though it seems I missed it in some other places)

Reply:
If we dropped support for it, then we need to tell Justin to remove any workflows for Node v14 from the list of required workflows.

<img width="824" alt="Screenshot 2023-11-14 at 10 26 22 AM" src="https://github.com/shakacode/react_on_rails/assets/8206741/2bc01309-a30f-407c-8a86-380dd73e9b5a">